contracts/staking/BaseShareField.sol

```diff
@@ -103,15 +103,14 @@ contract BaseShareField is DSMath, DAOContract {
 						1e18 -
 						userInfo.rewardDebt; // Divide 1e18 to reduce 18 Decimals since rewardDebt in 18 decimals
 				uint256 rewardPerBlock =
-					rdiv(pending * 1e9, blocksToPay * 1e27); // bring both variable to 27decimals since DSMath works on 27 decimals 
+					rdiv(pending , blocksToPay * 1e18); // bring both variable to 18 decimals so they would be in same decimals
 				pending =
 					rmul(
-						((firstMonthBlocksToPay * 50 * 1e25) +
+						((firstMonthBlocksToPay * 5 * 1e17) + // multiply with 1e17 since normally there is divide 10
```

Comment:
i would be verbose here firstMonthBlocksToPay  * 1e18 * 5 / 10 //multiply first month by 0.5x (5/10)